core/src/main/java/io/kestra/core/services/FlowService.java

```diff
@@ -83,131 +80,11 @@ private Stream<Flow> keepLastVersionCollector(Stream<Flow> stream) {
             .filter(Objects::nonNull);
     }
 
-    public List<FlowWithFlowTrigger> flowWithFlowTrigger(Stream<Flow> flowStream) {
-        return flowStream
-            .filter(flow -> flow.getTriggers() != null && flow.getTriggers().size() > 0)
-            .filter(flow -> !flow.isDisabled())
-            .flatMap(flow -> flow.getTriggers()
-                .stream()
-                .filter(abstractTrigger -> !abstractTrigger.isDisabled())
-                .map(trigger -> new FlowWithTrigger(flow, trigger))
-            )
-            .filter(f -> f.getTrigger() instanceof io.kestra.core.models.triggers.types.Flow)
-            .map(f -> new FlowWithFlowTrigger(
-                    f.getFlow(),
-                    (io.kestra.core.models.triggers.types.Flow) f.getTrigger()
-                )
-            )
-            .collect(Collectors.toList());
-    }
-
     protected boolean removeUnwanted(Flow f, Execution execution) {
         // we don't allow recursive
         return !f.uidWithoutRevision().equals(Flow.uidWithoutRevision(execution));
     }
```

Comment:
I think this method is only used on the new AbstractFlowTriggerService, if so, move it there